src/theory/quantifiers/expr_miner.cpp

```diff
@@ -91,6 +91,7 @@ void ExprMiner::initializeChecker(std::unique_ptr<SmtEngine>& checker,
       checker->setTimeLimit(options::sygusExprMinerCheckTimeout(), true);
       checker->setLogic(smt::currentSmtEngine()->getLogicInfo());
       checker->setOption("sygus-rr-synth-input", false);
+      checker->setOption("input-language", "smt2");
```

Comment:
Should we drop the condition on line 80? At the moment the behavior is not inuitive: if you set a timeout or the other option is set, we setup the proper defaults, otherwise we dont.